packages/rfw/lib/src/flutter/argument_decoders.dart

```diff
@@ -1420,3 +1447,57 @@ class ArgumentDecoders {
     }
   }
 }
+
+// TODO(shuoch-g): Move this class to other places where more appropriate.
+/// An implementation of [SliverPersistentHeaderDelegate] that takes in fixed
+/// child widget.
+/// 
+/// Caller can configure [maxExtent] and [minExtent], but the returned widget
+/// in [build] is always [child] which is fixed (doesn't shrink).
+class FixedSliverPersistentHeaderDelegate
+    extends SliverPersistentHeaderDelegate {
+  /// Creates a [FixedSliverPersistentHeaderDelegate] delegate.
+  /// 
+  /// The [fixedMaxExtent], [fixedMinExtent] and [child] are all required,
+  const FixedSliverPersistentHeaderDelegate({
+    required this.fixedMaxExtent,
+    required this.fixedMinExtent,
+    required this.child,
+  });
+
+  /// Fixed `maxExtent`.
+  /// 
+  /// In [build], the [child] is fixed (not shrink according to this value).
+  final double fixedMaxExtent;
+
+  /// Fixed `maxExtent`.
+  /// 
+  /// In [build], the [child] is fixed (not shrink according to this value).
+  final double fixedMinExtent;
+
+  /// Fixed widget to return in [build].
+  ///
+  /// In [build], it is fixed (not shrink).
```

Comment:
Not sure what "In [build], it is fixed (not shrink)." means

Reply:
updated the doc, is it clear now ?